app/editor/src/store/slices/workorder/interfaces/IWorkOrderListFilter.ts

```diff
@@ -0,0 +1,11 @@
+import { ISortBy } from 'features/content/list-view/interfaces';
+import { WorkOrderStatusName, WorkOrderTypeName } from 'hooks';
+
+export interface IWorkOrderListFilter {
```

Comment:
This filter seems specific to the WorkOrderList component.  Keep it close to that component the same way we are in the list-view.